setup.py

```diff
@@ -52,7 +52,14 @@ def get_version(filename):
     ]},
     packages=find_packages('src'),
     zip_safe=False,
-    install_requires=['pyyaml', 'six', 'requests', 'aws-sam-translator>=1.6.0', 'jsonpatch', 'jsonschema~=2.6.0', 'pathlib2'],
+    install_requires=[
+        'pyyaml',
+        'six~=1.11',
+        'requests>=2.15.0',
+        'aws-sam-translator>=1.8.0',
+        'jsonpatch', 'jsonschema~=2.6',
+        'pathlib2>=2.3.0;python_version<"3.6"'
```

Comment:
you can probably do `<"3.4"` unless you depend on `__fspath__` or similar like pytest does